docs/en/observability/monitor-servers.asciidoc

```diff
@@ -32,3 +32,115 @@ This metric relies on the same indices as the logs.
+
+Without leaving the <<view-infrastructure-metrics,*Inventory*>> page, you can view enhanced metrics relating to each host
+running in your infrastructure. On the waffle map, select the host to display the host metrics
+overlay.
+
+The enhanced metrics overlay contains the following tabs:
+
+// This is collapsed by default
+[%collapsible]
+.*Metrics*
+====
+
+[role="screenshot"]
+image::images/metrics-overlay.png[Host metrics]
+
+The *Metrics* tab displays CPU, load, memory, and network metrics relating to the host.
+You can change the time range to view metrics over the last 15 minutes, hour, 3 hours, 24 hours, or
+over the previous seven days. You can also hover over a specific time period on a chart to compare the various metrics
+at that given time.
+
+|=== 
+
+| *CPU* | Averages of `system.cpu.user.pct` divided by `system.cpu.cores` and `system.cpu.system.pct` divided by `system.cpu.cores`.
+
+| *Load* | Averages of `system.load.1`, `system.load.5`, and `system.load.15`.
+
+| *Memory* | For Linux systems, memory used is the average of `system.memory.actual.used.bytes` and memory free is the average
+of `system.memory.actual.free`. For non-Linux systems, memory used is the average of `system.memory.used.bytes` and memory free is the average
+of `system.memory.free`.
+
+| *Network* | Rates of `system.network.in.bytes` and `system.network.out.bytes`.
+
+|=== 
+====
+
+[%collapsible]
+.*Logs*
+====
+
+[role="screenshot"]
+image::images/logs-overlay.png[Host logs]
+
+The *Logs* tab displays logs relating to the host that you have selected. By default, the logs tab displays the following columns. 
+
+|=== 
+
+| *Timestamp* | The timestamp of the log entry from the `timestamp` field. 
+
+| *Message* | The message extracted from the document.
+The content of this field depends on the type of log message.
+If no special log message type is detected, the {ecs-ref}/ecs-base.html[Elastic Common Schema (ECS)]
+base field, `message`, is used.
+
+|=== 
+
+You can customize the logs view by adding a column for an arbitrary field you would like
+to filter by. For more information, see <<customize-stream-page,Customize Stream>>.
+To view the logs in the {logs-app} for a detailed analysis, click *Open in Logs*.
+====
+
+[%collapsible]
+.*Processes*
+====
+
+[role="screenshot"]
+image::images/processes-overlay.png[Host processes]
+
+The *Processes* tab lists the total number of processes (`system.process.summary.total`) running on the host,
+along with the total number of processes in these various states:
+
+* Running (`system.process.summary.running`)
+* Sleeping (`system.process.summary.sleeping`)
+* Stopped (`system.process.summary.stopped`)
+* Idle (`system.process.summary.idle`)
+* Dead (`system.process.summary.dead`)
+* Zombie (`system.process.summary.zombie`)
+* Unknown (`system.process.summary.unknown`)
+
+The processes listed in the *Top processes* table are based on an aggregation of the top CPU and the top memory consuming processes.
```

Comment:
Here we should add that the number of Top processes is controlled via the `process.include_top_n.by_cpu` and `process.include_top_n.by_memory`, see https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-metricset-system-process.html#_configuration_11

cc @phillipb can you please confirm this part?

Reply:
Yep, this is correct

Reply:
Thanks, I'll add this.